samples/bookinfo/src/productpage/productpage.py

```diff
@@ -80,18 +87,69 @@
     "reviews" : reviews,
 }
 
+# A very basic OpenTracing tracer (with null reporter)
+tracer = Tracer(
+    one_span_per_rpc=True,
+    service_name='productpage',
+    reporter=NullReporter(),
```

Comment:
Haven't used the python client, but assume that if the reporter is not specified, it reports the spans to the agent? Would it not be better to use that so the spans are included in the trace?

Reply:
This is kind of a minimal viable first pass. It doesn't introduce any new functionality eg the ability to actually do anything other than propagate headers. I see it as basically a starting point.

Reply:
I like this approach but also think that it would be good to explain it clearly with a few comments in the code. It should say that the purpose here is to simply forward headers, which is the Istio requirement for trace propagation, but that it's being done in a way that sets things up to easily add more application-specific tracing later. We don't want people to think that there is any more work that they need ti do to enable Istio tracing than there actually is.

Reply:
Agreed